### PR TITLE
feat: create portfolio and testimonial

### PR DESCRIPTION
**Pull request by luan-thanh**

### CODE REVIEW

File: controllers/home_controller.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
+const Portfolio = require('../models/portfolio');
+const Testimonial = require('../models/testimonial');
+
 async function index(req, res, next) {
```
Comment (Luan-Thanhh) [inline controllers/home_controller.js:4]: Oce

**Review by luan-thnh — APPROVED** [review]

File: controllers/portfolio_controller.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const Portfolio = require('../models/portfolio');
+
```
Comment (Luan-Thanhh) [inline controllers/portfolio_controller.js:2]: Oce

**Review by luan-thnh — APPROVED** [review]

File: controllers/home_controller.js [hunk 3]
```diff
@@ -1,7 +1,15 @@
+const Portfolio = require('../models/portfolio');
+const Testimonial = require('../models/testimonial');
```
Comment (Luan-Thanhh) [inline controllers/home_controller.js:2]: Oce

**Review by luan-thnh — APPROVED** [review]

**Review by luan-thnh — APPROVED** [review]

**Comment by luan-thanh** [comment]
Oce

File: controllers/home_controller.js [hunk 4]
```diff
@@ -1,7 +1,15 @@
+const Portfolio = require('../models/portfolio');
```
Comment (luan-thanh) [inline controllers/home_controller.js:1]: Test

Comment (luan-thnh) [reply controllers/home_controller.js:1]: Oce

**Review by luan-thnh — CHANGES_REQUESTED** [review]

**Review by luan-thnh — APPROVED** [review]

**Pull request merged**